src/Server.hpp

```diff
@@ -46,10 +46,11 @@ class Server : public afsm::def::state_machine<Server> {
 
         // @formatter:off
         using transitions = transition_table <
-        // Start            Event                               Next            Action
+        // Start            Event                               Next            Action                                                               Guard
```

Comment:
Der Moment, in dem Jonas viele Leerzeichen hinzufügt, muss mit einem Kommentar festgehalten werden :D